docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -25,13 +25,14 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
 | 2.1.8xx          | 16.2 (No VS)       | July '19     | Aug '21   |
 | 3.1.1xx          | 16.4               | Dec '19      | Oct '21   |
 | 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
-| 5.0.1xx          | 16.8               | November '20 | Mar '21   |
+| 5.0.1xx          | 16.8               | Nov '20      | Mar '21   |
 | 5.0.2xx          | 16.9               | March '21    | Aug '22   |
 | 5.0.3xx          | 16.10              | May '21      | Aug '21   |
 | 5.0.4xx          | 16.11              | Aug '21      | Feb '22*  |
-| 6.0.1xx          | 17.0**             | Nov. '21     | Jul '23   |
-| 6.0.2xx          | 17.1               |  '21         |           |
-| 6.0.3xx          | 17.2               |  '21         |           |
+| 6.0.100-rc1      | 17.0-preview 4     | Sep '21      | N/A       |
+| 6.0.100          | 17.0**             | Nov '21      | May '23   |
```

Comment:
For these three, please remove the May dates. Those are all dependent on when 17.2 ships and we have not committed to a specific date yet (I wouldn't want to publish something and then have it move).

Reply:
done, I replaced them with tbd.